bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/reconciler/AbstractReconciler.java

```diff
@@ -449,7 +471,7 @@ public void install(ITextViewer textViewer) {
 		synchronized (this) {
 			if (fThread != null)
 				return;
-			fThread= new BackgroundThread(getClass().getName());
+			fThread= new BackgroundThread(getClass().getName() + " reconciler thread"); //$NON-NLS-1$
```

Comment:
I believe we should better undo this change, just keep the class name as before, additional text it doesn't bring much and in JDT case makes the job name definitely not better (beside the fact that this is a "job" now):

![image](https://github.com/user-attachments/assets/b5689c63-d746-4952-a661-35ec74b99686)

Reply:
Yes, just like it was before.